develop/reference/articles/12-project-templates/19-npm-vuejs-portlet-template.markdown

```diff
@@ -57,14 +57,17 @@ or
         -DpackageJsonVersion=1.0.0 \
         -DliferayVersion=7.0
 
+**Important:** This sample works for Liferay DXP 7.0 Fix Pack 44+ and Liferay Portal
+CE GA7+.
```

Comment:
Hey @Claudioss23 , the ticket only mentions to add that the sample works for  Liferay DXP Fix Pack 44+, do you know if it also works for Liferay Portal CE GA7+. (I know that this is how it looks on the other doc, but this could be different :thinking: )

Reply:
Yes, because Fix Pack 44+ is contained in 7.0 Portal CE GA7+ versions.

Reply:
Ok, good to know!